streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/StreamToTableJoinScalaIntegrationTestImplicitSerdes.scala

```diff
@@ -152,19 +158,14 @@ class StreamToTableJoinScalaIntegrationTestImplicitSerdes extends StreamToTableJ
     // Change the stream from <user> -> <region, clicks> to <region> -> <clicks>
     val clicksByRegion: KStreamJ[String, JLong] = userClicksJoinRegion
       .map {
-        new KeyValueMapper[String, (String, JLong), KeyValue[String, JLong]] {
-          def apply(k: String, regionWithClicks: (String, JLong)) =
-            new KeyValue[String, JLong](regionWithClicks._1, regionWithClicks._2)
-        }
+        (_: String, regionWithClicks: (String, JLong)) => new KeyValue[String, JLong](regionWithClicks._1, regionWithClicks._2)
```

Comment:
I have a vaugue recollection that using SAM conversion may cause the java 8/scala 2.11 build to fail. We'll have to see...

Reply:
Yep, it did. I've rolled back these changes.